Src/Sarif.PatternMatcher.Sdk/FingerprintElements.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk
+{
+    [Flags]
+    internal enum FingerprintElements
```

Comment:
>FingerprintElements [](http://example.com/codeflow?start=18&length=19)

FYI, this code is being added preparatory to a future change. It is not consume currently, nor is it in a finished condition (requires doc comments, for one thing).